source/internal/_isFunction.js

```diff
@@ -1,3 +1,6 @@
 export default function _isFunction(x) {
-  return Object.prototype.toString.call(x) === '[object Function]';
+  var type = Object.prototype.toString.call(x);
+  return type  === '[object Function]' ||
+    type === '[object AsyncFunction]' ||
+    type === '[object GeneratorFunction]';
```

Comment:
Does not support `AsyncGeneratorFunction`.

Reply:
I don’t mind adding that if you’re certain that’s the exact string of the extra case I need to add.

Reply:
I don’t mind adding that if you’re certain that’s the exact string of the extra case I need to add.

Reply:
`"[object AsyncGeneratorFunction]"`

Reply:
Will update! @CrossEye, please speak up if this change would prevent having this merged.

Reply:
updated